workflow/comp_pred_ridge_and_LSTM.ipynb

```diff
@@ -34,20 +34,9 @@
   },
```

Comment:
<p>Interesting to see that the model has found a completely different fit now vs. previously. Again it's just (mostly) predicting the mean value 🤷</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>